servicebus-parametrised.tf

```diff
@@ -1,13 +1,13 @@
-module "servicebus-namespace-premium" {
+module "servicebus-namespace" {
   providers = {
     azurerm.private_endpoint = azurerm.private_endpoint
   }
   source              = "git@github.com:hmcts/terraform-module-servicebus-namespace?ref=master"
-  name                = "${var.product}-${var.env}-premium"
+  name                = "${var.product}-${var.env}"
   resource_group_name = azurerm_resource_group.rg.name
   location            = var.location
   env                 = var.env
   common_tags         = var.common_tags
-  zone_redundant      = true
-  sku                 = "Premium"
-}
\ No newline at end of file
+  zone_redundant      = var.servicebus_zone_redundant #currently unused, we don't think we need redundancy currently but could change.
```

Comment:
to meet HMCTS resiliency requirements production services should all be ran across multiple available zones

Reply:
Switched back to redundant setup